app/scripts/directives/pickadate.js

```diff
@@ -34,12 +36,56 @@ angular.module('confRegistrationWebApp')
         //initialize datepicker
         scope.picker = $(element).find('input').first().pickadate(pickerOptions).pickadate('picker');
 
+        //function to set min date
+        scope.setMinDate = function(){
+          if(!angular.isUndefined(scope.minDate) && scope.minDate !== ''){
+            var dateArray = scope.minDate.split('-');
+            dateArray[1] = dateArray[1] - 1;
+            scope.picker.set('min',dateArray);
+          }
+        };
+        
+        //function to set max date
+        scope.setMaxDate = function(){
+          if(!angular.isUndefined(scope.maxDate) && scope.maxDate !== ''){
+            var dateArray = scope.maxDate.split('-');
+            dateArray[1] = dateArray[1] - 1;
+            scope.picker.set('max',dateArray);
+          }
+        };
+		
+        //set min and max date
+        scope.setMinDate();
+        scope.setMaxDate();
+
+        scope.$watch('minDate', function (newMinDate, oldMinDate) {
+          if (angular.isUndefined(newMinDate) || newMinDate === '' || angular.equals(newMinDate, oldMinDate)) {
```

Comment:
What case is the `angular.equals(newMinDate, oldMinDate)` handling? I would think that would never be true as `$watch` shouldn't call the function if there are no changes...